addons/test_mail/tests/test_mail_message.py

```diff
@@ -373,3 +373,21 @@ def test_message_link_by_employee(self):
             self.assertEqual(res.url, expected_url)
         with self.subTest(deleted_message=deleted_message):
             res = self.url_open(f'/mail/message/{deleted_message.id}')
+
+@tagged("mail_message")
+class TestMessageFailure(MailCommon, HttpCase):
+
+    def test_failure_thread_display_name_as_name(self):
+        unnamed_record = self.env['mail.test.simple.unnamed'].create({'description': 'Some description'})
+        msg = unnamed_record.message_post(body='Some body', author_id=self.partner_employee.id)
+        self.env['mail.notification'].create({
```

Comment:
That test makes no sense: why is the author having a failed notification ?

Reply:
I was a bit confused too, but the failure is shown on the sender not receiver so this is actually good to check that failure is on author.

I added another partner with invalid email and made the `user_employee` send the message, then check failure on `user_employee` PoV, which is the right business flow.

Reply:
Well there are some strange things, but as it is the freeze (aka not much time) and we have to ship the fix (aka not much time), let us say it is ok ^^ main thing is that the bug itself is covered :) . Cheers !